romio.spec

```diff
@@ -25,23 +24,22 @@ URL:        http://www.mpich.org/
 %endif
 %endif
 %endif
-Source0:    https://build.hpdd.intel.com/job/daos-stack/job/mpich/job/daos_adio-rpm/lastSuccessfulBuild/artifact/artifacts/%{distro}/%{name}-%{version}.tar.gz
-Patch0:     packaged-runtests-%{distro}.patch
```

Comment:
Looking at the changes in src/mpi/romio/test/runtests.in in https://github.com/pmodels/mpich/compare/v3.3...v3.4a2 it's not surprising that this patch doesn't apply cleanly any more.  But I suspect the parts of the patch that add `$testsdir` is still going to be necessary.  Although I don't recall what the relevance of that runtests file is in our workflow.  Maybe it was just being patched to be able to run it to validate the packaging build and should likely be added to a `%check` section of the `romio.spec` at some point in the future.

In any case, regenerating this patch to apply to the current `runtests` shouldn't be difficult.  It should just be a matter of unpacking the new romio tarball, applying the patch, fixing rejects and creating a new patch, either by diffing the unpacked tarball to another unpacked pristine tarball, or by just taking a copy of the file being patched before patching and fixing it.